mdbenchmark/analyze.py

```diff
@@ -132,15 +133,31 @@ def plot_analysis(df, ncores):
     help='Number of cores per node. If not given it will be parsed from the '
     'benchmarks log file.',
     show_default=True)
-def analyze(directory, plot, ncores):
+@click.option(
+    '-o',
+    '--output_name',
+    help="Name of the output .csv file.",
+    default="runtime.csv",
```

Comment:
Do we want to keep it this way? I initially thought we will use the `host`, `module` and `gpu` values for the output file name? This way we could prepare for #24, where we would specify different columns to group by. These be used for the filenames..? Or am I overthinking this?

Reply:
you are overthinking this.